7_drivers_munge/src/GCM_driver_utils.R

```diff
@@ -374,7 +385,8 @@ map_missing_cells <- function(out_file, lake_cell_tile_xwalk, cell_info, grid_ce
           legend.position="top") + 
     ggtitle("What cells are missing driver data?", 
             subtitle = paste(
-              sprintf("%s cells are being used to fill in missing driver data for %s cells", length(cells_being_used), length(gcm_cells_w_o_data)),
+              sprintf("%s queried cells (hollow, light grey border) returned data", length(grid_cells_w_data)),
```

Comment:
Might reword to make it clear what we mean by "returned data" 
```suggestion
              sprintf("%s queried cells (hollow, light grey border) returned data and have lakes present ", length(grid_cells_w_data)),
```